extension/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "9.7.0",
   "description": "Integration between commercetools and Adyen payment service provider",
   "license": "MIT",
+  "type": "module",
   "scripts": {
     "check-coverage": "nyc check-coverage --lines 89",
     "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
     "cover:unit": "nyc --silent npm run unit",
     "cover:integration": "nyc --silent --no-clean npm run integration",
     "cover:e2e": "nyc --silent --no-clean npm run e2e",
-    "unit": "nyc mocha --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.js\"",
-    "integration": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/integration/**/*.spec.js\"",
+    "unit": "nyc mocha --experimental-json-modules --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.{cjs,js}\"",
+    "integration": "nyc mocha --experimental-json-modules --file ./test/initial-integration-test-set-up.js \"test/integration/**/*.spec.js\"",
     "e2e": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/e2e/**/*.spec.js\"",
     "start": "node ./src/init.js",
-    "lint": "eslint ./src ./test",
```

Comment:
Maybe you can leave it as it is? https://github.com/commercetools/commercetools-subscriptions/blob/2a778859c04d6455ae8e6ccbf295ec7edac5813e/package.json#L18

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/929/commits/68cecc94cd71470798fc355f702c2a47dfb374a2

Along with this fix, I had to fix all the lint issues.